content/en/news/support/advance-notice-opencensus-deprecation/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Advance notice of Istio deprecating OpenCensus
+subtitle: Support Announcement
+description: Advance notice of Istio deprecating OpenCensus.
+publishdate: 2023-08-17
+---
+
+OpenCensus has [sunsetted](https://opentelemetry.io/blog/2023/sunsetting-opencensus) on July 31st, 2023.
+
+The OpenCensus sunsetting will impact Istio's usage of OpenCensus in tracing, logging, and metrics. Istio
```

Comment:
Can you expand on where it's used exactly? Is it just Stackdriver and OpenCensus providers? Do we export zipkin with OpenCensus?

Reply:
Just Stackdriver and OpenCensus providers. We do not export zipkin with OpenCensus.